lte/gateway/c/oai/tasks/sctp/sctpd_uplink_server.cpp

```diff
@@ -107,9 +111,11 @@ Status SctpdUplinkImpl::NewAssoc(
 
 Status SctpdUplinkImpl::CloseAssoc(
     ServerContext* context, const CloseAssocReq* req, CloseAssocRes* res) {
+  uint32_t ppid;
   uint32_t assoc_id;
   bool reset;
 
+  ppid	   = req->ppid();
```

Comment:
why is this not used in sctp_itti_send_com_down_ind on line 122? Since the function signature is modified, this should not have compiled successfull.